lib/worker.js

```diff
@@ -18,6 +18,14 @@ const data = Buffer.from(dataBuf)
 async function start () {
   let fn
   try {
+    if(typeof process !== "undefined"){
+      //reused the error code.
+      const err = new Error()
```

Comment:
```suggestion
      const err = Error()
```

Also, an appropriate message should be passed to the constructor.